charts/kubeclarity/values.yaml

```diff
@@ -341,3 +336,15 @@ kubeclarity-postgresql:
 
 # End of KubeClarity Postgres Values
 #######################################################################################
+
+#######################################################################################
+## KubeClarity Postgres Authentication Values
+kubeclarity-postgresql-auth:
```

Comment:
can you explain why you extracted the values from `kubeclarity-postgresql.auth`?

Reply:
My idea was that kubeclarity-postgresql was used for the bitnami postgresql instance deployed when `kubeclarity-postgresql.enabled` boolean is true.

I wanted to extract that because `kubeclarity-postgresql-auth` mapping would configure both the external postgresql or the internal one (the one deployed in the helm dependencies).

As there can only be one database, I wanted to mutualize that.

Reply:
but now `existingSecret` and `database` can't be configured for the bitnami postgresql instance